<ui version="4.0" >
 <class>myQtAppDLG</class>
 <widget class="QWidget" name="myQtAppDLG" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>390</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>390</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>My first Qt GUI App</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Number 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Number 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>0</hsizetype>
     <vsizetype>0</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_do" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>250</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Do something</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_about" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox2" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>80</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>120</y>
     <width>351</width>
     <height>121</height>
    </rect>
   </property>
   <property name="html" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:MS Shell Dlg 2; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:MS; font-size:8pt;">this is textEdit&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox1" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>50</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>50</y>
     <width>70</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>Yes or No</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_browse" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>0</hsizetype>
     <vsizetype>0</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Browse..</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>Apple</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Banana</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Orange</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Whatever</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>10</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>